source/data/Symbol_info.rst

```diff
@@ -6,95 +6,156 @@
+| ``<Base Currency><Quote Currency>``   | ``EURUSD`` — Euro to US Dollar             |
+|                                       |                                            |
+|                                       | ``USDGBP`` — US Dollar to British Pound    |
++---------------------------------------+--------------------------------------------+
 	
 Futures
 ~~~~~~~~
-``<Exchange>:<Root><2 Digit Day(optional)><Month Code><4 Digit Year>``
-
-.. code-block:: cfg
 
-	IMH2 → LIFFE:IMH2012
-	MAH2 → EUIDX:MAH2012
-	AVU2 → EUREX:AVU2012
-
-If more than one contract is expired in one month, the expiration day is added to the name after the root.
++-----------------------------------------------------+--------------------------------------------------------------------------+
+| Type                                                | Format                                                                   |
++=====================================================+==========================================================================+
+| Standard                                            | ``<Symbol Root><Month Code><Four-digit Year>``                           |
+|                                                     |                                                                          |
+|                                                     |                                                                          |
++-----------------------------------------------------+--------------------------------------------------------------------------+
+| When more than one contract is expired in one month | ``<Symbol Root><Two-digit Expiration Day><Month Code><Four-digit Year>`` |
+|                                                     |                                                                          |
+|                                                     |                                                                          |
++-----------------------------------------------------+--------------------------------------------------------------------------+
+
+The table below represents months and their corresponding codes:
+
++-----------+------------+
+| Month     | Month Code |
++===========+============+
+| January   | F          |
++-----------+------------+
+| February  | G          |
++-----------+------------+
+| March     | H          |
++-----------+------------+
+| April     | J          |
++-----------+------------+
+| May       | K          |
++-----------+------------+
+| June      | M          |
++-----------+------------+
+| July      | N          |
++-----------+------------+
+| August    | Q          |
++-----------+------------+
+| September | U          |
++-----------+------------+
+| October   | V          |
++-----------+------------+
+| November  | X          |
++-----------+------------+
+| December  | Z          |
++-----------+------------+
+
+The corresponding examples are added below:
```

Comment:
`Corresponding` confused me in this case. It took me some time to guess what these examples correspond to, as the examples are far from the table. E.g., it's ok in the L89 as both words are in the same sentence.
```suggestion
The examples of future types are added below:
```
or
```suggestion
Consider the examples below:
```